templates/taxi/driver_list.html

```diff
@@ -1,17 +1,18 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Driver list</h1>
+  <h1 class="taxi-title">Drivers</h1>
   {% if driver_list %}
-    <ul>
-      {% for driver in driver_list%}
+    <ul class="driver-list">
+    {% for driver in driver_list %}
+      <a href="{{ driver.get_absolute_url }}" class="driver-item">
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          <p class="driver-info">ID: {{ driver.id }}</p>
+          <p class="driver-info">Name: {{ driver.first_name }}</p>
+          <p class="driver-info">Surname: {{ driver.last_name }}</p>
         </li>
-      {% endfor %}
+      </a>
```

Comment:
Placing the `<a>` tag around the `<li>` element is not semantically correct. Instead, consider placing the `<a>` tag inside the `<li>` or wrapping the entire `<li>` content with the `<a>` tag, but not the `<li>` itself.